core/server/worker/src/test/java/alluxio/worker/block/BlockMasterClientTest.java

```diff
@@ -80,23 +79,26 @@
 
 public class BlockMasterClientTest {
 
+  // alert: you CANNOT just set TEST_SOCKET_ADDRESS to
+  // configuration, because it will get turned to string "localhost/127.0.0.1:9999"
+  // This string cannot be parsed accurately by the current parsing algorithm
   private static final InetSocketAddress TEST_SOCKET_ADDRESS =
-      new InetSocketAddress("0.0.0.0", 9999);
+      new InetSocketAddress("localhost", 9999);
+  private static final String TEST_SOCKET_ADDRESS_STRING = "localhost:9999";
 
   @Rule
   public ConfigurationRule mConfiguration = new ConfigurationRule(ImmutableMap
-      .of(PropertyKey.MASTER_HOSTNAME, "localhost",
-          PropertyKey.MASTER_RPC_PORT, 9999,
-          PropertyKey.MASTER_RPC_ADDRESSES, ImmutableList.of(TEST_SOCKET_ADDRESS),
+      .of(
+          PropertyKey.MASTER_RPC_ADDRESSES, ImmutableList.of(TEST_SOCKET_ADDRESS_STRING),
           PropertyKey.USER_RPC_RETRY_MAX_DURATION, "5s",
           PropertyKey.USER_RPC_RETRY_BASE_SLEEP_MS, "100ms",
           PropertyKey.USER_RPC_RETRY_MAX_SLEEP_MS, "500ms",
-          PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.NOSASL),
+          PropertyKey.SECURITY_AUTHENTICATION_TYPE, AuthType.NOSASL,
+          PropertyKey.USER_CONF_CLUSTER_DEFAULT_ENABLED, false),
```

Comment:
Add a comment here that explains why cluster defaults cannot be acquired.

Reply:
Done